stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/info/shared/files/ChatInfoSharedFilesFragment.kt

```diff
@@ -50,13 +51,7 @@ class ChatInfoSharedFilesFragment : Fragment() {
         binding.filesRecyclerView.adapter = adapter
         binding.filesRecyclerView.addOnScrollListener(scrollListener)
         adapter.setAttachmentClickListener { attachmentItem ->
-            attachmentItem.attachmentWithDate.attachment.url?.let { url ->
-                startActivity(
-                    Intent.makeMainSelectorActivity(Intent.ACTION_MAIN, Intent.CATEGORY_APP_BROWSER).apply {
-                        data = Uri.parse(url)
-                    }
-                )
-            }
+            ChatUI.instance().navigator.navigate(AttachmentDestination(Message(), attachmentItem.attachmentWithDate.attachment, requireContext()))
```

Comment:
`Message` is not really needed for file attachments, only for images, so passing a new one is enough for now.
BTW, Attachments endpoint will be updated to not only obtain attachments but the full message as well